### ISSUE TITLE
MacOs - Icon doesn't transition from dark to light

### ISSUE THREAD

**Issue by rtpHarry** [ActivityWatch/activitywatch#358]
**For us to be able to understand the issue you're having you need to follow this template.** Please make sure there isn't already an issue for the bug you've found, otherwise it'll just take more resources, which means less time for us to actually fix bugs. Thanks for reporting!

## Describe the bug
I'm running the custom build from #334 and im excited that it's up and running now on my system.

I have the MacOS setting to automatically turn to dark mode in the evening, and back in the day.

Last night it was on dark mode, this morning I've come back and the tray icon hasn't updated, it's still white:

![image](https://user-images.githubusercontent.com/1038062/75668004-807d2880-5c70-11ea-9160-8c5a214aa920.png)

## To Reproduce
1. Run ActivityWatch with MacOS on night mode
1. Wait until system goes back to day mode
1. Observe icon colour doesnt change

## Expected behavior
Icon should turn black in day mode.

## Environment
 - ActivityWatch version:Version: v0.8.dev+c6433ea
 - OS: macOS Catalina

**Comment by ErikBjare** [comment]
Ah yes, the change that adds support for dark/light macOS theme was added in https://github.com/ActivityWatch/aw-qt/pull/52

It looks to me like it wasn't done the proper way however. We added two icons and detected the theme at startup, while some suggest we should only have one and then use `myQIcon.setIsMask(True)` as described [here](https://stackoverflow.com/a/41903980/965332).

Want to take a look at it @xylix?
